spec/system/admin/admin_bans_or_warns_user_spec.rb

```diff
@@ -62,7 +62,7 @@ def unsuspend_user
 
     expect(user.suspended?).to eq(true)
     expect(user.trusted).to eq(false)
```

Comment:
With all of these refactors, are we also planning on changing `trusted` to `trusted?`

Reply:
maybe we already did (and it's just a change in the callers)?

https://github.com/forem/forem/blob/main/app/models/user.rb#L384

Reply:
I noticed that, @djuber 👀